demo/splunk/wazuh_cf_splunk.sh

```diff
@@ -11,8 +11,8 @@ eth0_ip=$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2  | cut -d' ' -f1
 wazuh_api_user=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiAdminUsername:' | cut -d' ' -f2)
 wazuh_api_password=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiAdminPassword:' | cut -d' ' -f2)
 wazuh_api_port=$(cat /tmp/wazuh_cf_settings | grep '^WazuhApiPort:' | cut -d' ' -f2)
-TAG="v3.12.3"
-APP_TAG="v3.12.3-7.3.4"
+TAG="v3.13.0"
+APP_TAG="v3.13.0-7.3.4"
```

Comment:
Splunk is now `v3.13.0-7.3.5`

https://github.com/wazuh/wazuh-splunk/tree/v3.13.0-7.3.5

Reply:
Check related installers also please

Reply:
Done on 2614fc00c1b76f2b0352c962e089adcf82bd0514. Thanks !

Reply:
Hi just noticed that the Splunk app link is not updated, check https://github.com/wazuh/wazuh-cloudformation/blob/2614fc00c1b76f2b0352c962e089adcf82bd0514/demo/splunk/wazuh_cf_splunk.sh#L27

You can download it from `https://wazuh-demo.s3-us-west-1.amazonaws.com/splunk-7.3.5-86fd62efc3d7-linux-2.6-x86_64.rpm`

Reply:
Updated on e855adbea797a36e5dbf555f02fccd1d43936f04.